### ISSUE TITLE
multiple ZEDs looking at same object - merge point cloud - configuring camera view

### ISSUE THREAD

**Issue by manikandtan-ck** [stereolabs/zed-examples#47]
I  have recorded multiple SVOs of the same object placed at the center of the field-of-view from 0,90,180 and 270 degrees. I wish to merge the 4 point cloud data together. 

![3d_0](https://user-images.githubusercontent.com/4781823/37098660-706eb69c-2244-11e8-89ea-c0b49e3ecfd6.jpg)
![3d_90](https://user-images.githubusercontent.com/4781823/37098661-710ec538-2244-11e8-9031-2bfb001ea303.png)
![3d_180](https://user-images.githubusercontent.com/4781823/37098663-718db97e-2244-11e8-893e-e76ffe62e10d.png)
![3d_270](https://user-images.githubusercontent.com/4781823/37098665-72349ad2-2244-11e8-9223-a09bd1a742e1.png)

I saw a reference to Rotation class but it seems to be for positional mapping.

How to configure the two cameras to render to the same 3D view?
Also, How to set the x,y,z coordinates of multiple cameras in a given 3d space?

**Comment by WASCHMASCHINE** [comment]
First of all, your depth maps look **really** bad.

Second, this should be mostly agnostic to the ZED SDK. 
You have four cameras with known poses, each gives you an RGB image and a depth image. The first camera can be your origin. You map those cameras in 3D according to your measured poses and display RGB+D in OpenGL concurrently.

Or you take snapshots of the point cloud and rearrange them manually in a point cloud viewer according to the camera poses.

So in short: You need to program it yourself to render to the same view, as you need the relative transformations. Also, xyz is not enough: You also need the orientation of the cameras.
